pkg/virtual/initializingworkspaces/builder/build.go

```diff
@@ -295,10 +296,10 @@ func BuildVirtualWorkspace(
 		}),
 	}
 
-	return map[string]framework.VirtualWorkspace{
-		wildcardWorkspacesName: wildcardWorkspaces,
-		workspacesName:         workspaces,
-		workspaceContentName:   workspaceContent,
+	return []rootapiserver.NamedVirtualWorkspace{
```

Comment:
is the order given by the fact that we store workspaces in an array?

Reply:
yes. First comes first serves.